### PR TITLE
import account.

### PR DESCRIPTION
**Pull request by Stumble**

### CODE REVIEW

File: x/account/manager/manager.go [hunk 1]
```diff
@@ -623,25 +513,62 @@ func (accManager AccountManager) GetAllGrantPubKeys(ctx sdk.Context, username li
...
+		totalSaving = totalSaving.Plus(v.Reward.UnclaimReward)
+		bank := &model.AccountBank{
+			Saving:          totalSaving,
+			FrozenMoneyList: v.Bank.FrozenMoneyList,
+			PubKey:          pubkey,
+			Sequence:        v.Meta.Sequence,
+			Username:        v.Username,
+		}
+		accManager.storage.SetInfo(ctx, v.Username, info)
+		accManager.storage.SetBank(ctx, addr, bank)
```
Comment (xiaoxiaff) [inline x/account/manager/manager.go]: Prefer get bank first to prevent conflict reset key.

File: x/developer/manager/manager.go [hunk 2]
```diff
@@ -325,6 +325,9 @@ func (dm DeveloperManager) BurnIDA(ctx sdk.Context, app, user linotypes.AccountK
 		return linotypes.NewCoinFromInt64(0), types.ErrBurnZeroIDA()
 	}
 	pool := dm.storage.GetReservePool(ctx)
+	if !pool.Total.IsGTE(bought) {
+		return linotypes.NewCoinFromInt64(0), types.ErrInsuffientReservePool()
+	}
 	pool.Total = pool.Total.Minus(bought)
 	pool.TotalMiniDollar = pool.TotalMiniDollar.Minus(used)
```
Comment (xiaoxiaff) [inline x/developer/manager/manager.go]: Is it possible TotalMiniDollar is insufficient to pay for the used?

Comment (Stumble) [reply x/developer/manager/manager.go]: nope, only coin may be insufficient.

File: x/post/types/events.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package types
+
+import (
+	linotypes "github.com/lino-network/lino/types"
+)
+
+// RewardEventV1 - used only for importing.
```
Comment (xiaoxiaff) [inline x/post/types/events.go]: is it possible to export without Original and Friction fields so v1 can be removed?

Comment (Stumble) [reply x/post/types/events.go]: The problem is the type of `	Evaluate   linotypes.Coin       `json:"evaluate"`  changed, and in v1 we don't have minidollar.

**Pull request merged**